typescript/src/lib/networks/network_context.ts

```diff
@@ -620,4 +653,113 @@ export class NetworkContext {
 
         return ok(null);
     }
+
+    // Docs available at https://docs.kurtosistech.com/kurtosis-client/lib-documentation
+    public async getServices(): Promise<Result<ServiceID[], Error>> {
```

Comment:
Typescript DOES have sets, so you can make the return type here `Set<ServiceID>`